### PR TITLE
addressing bugs #174, #184, #187

### PR DESCRIPTION
**Pull request by JaseZiv**

### CODE REVIEW

File: R/get_match_report.R [hunk 1]
```diff
@@ -22,7 +22,7 @@
     game <- each_game_page %>% rvest::html_nodes("h1") %>% rvest::html_text()
 
     # tryCatch( {League <- each_game_page %>% rvest::html_nodes("h1+ div a") %>% rvest::html_text()}, error = function(e) {League <- NA})
-    tryCatch( {League <- each_game_page %>% rvest::html_nodes("h1+ div a") %>% rvest::html_text()}, error = function(e) {League <- NA})
+    tryCatch( {League <- each_game_page %>% rvest::html_nodes("h1+ div a:nth-child(1)") %>% rvest::html_text()}, error = function(e) {League <- NA})
```
Comment (JaseZiv) [inline R/get_match_report.R:25]: Was causing issues when there were multiple legs of cup ties with downstream functions that called this internal function

File: R/internals.R [hunk 2]
```diff
@@ -197,7 +197,7 @@
     gsub("_Var", "", .) %>%
     gsub("#", "Num_", .) %>%
     gsub("%", "_percent", .) %>%
-    gsub("_Performance", "", .) %>%
+    # gsub("_Performance", "", .) %>%
```
Comment (JaseZiv) [inline R/internals.R:200]: for `fb_team_match_log_stats()`, when `stat_type='keeper'`, 'GA' was being returned as a column twice (both for the team and individual). Need to include '_Performance' to distinguish the two

File: R/tm_squad_stats.R [hunk 3]
```diff
@@ -24,9 +24,9 @@ tm_squad_stats <- function(team_url) {
 
     if(!is.na(team_data_page)) {
 
-      team_name <- team_data_page %>% rvest::html_nodes("h1") %>% rvest::html_text() %>% stringr::str_squish()
-      league <- team_data_page %>% rvest::html_nodes(".hauptpunkt a") %>% rvest::html_text() %>% stringr::str_squish()
-      country <- team_data_page %>% rvest::html_nodes(".mediumpunkt img") %>% rvest::html_attr("title")
+      team_name <- team_data_page %>% rvest::html_nodes(".data-header__headline-wrapper--oswald") %>% rvest::html_text() %>% stringr::str_squish()
```
Comment (JaseZiv) [inline R/tm_squad_stats.R:27]: Transfermarkt just changed some of their html

File: tests/testthat/test-fbref.R [hunk 4]
```diff
@@ -298,7 +298,7 @@ test_that("fb_team_player_stats() works", {
 Sys.sleep(3)
 test_that("fb_team_match_log_stats() works", {
   testthat::skip_on_cran()
-  liv_passing_log <- fb_team_match_log_stats(team_url= "https://fbref.com/en/squads/822bd0ba/Liverpool-Stats", stat_type= 'passing', time_pause = 4)
+  liv_passing_log <- fb_team_match_log_stats(team_url= "https://fbref.com/en/squads/822bd0ba/Liverpool-Stats", stat_type= 'keeper', time_pause = 4)
```
Comment (tonyelhabr) [inline tests/testthat/test-fbref.R:301]: maybe change the variable name to `liv_keeper_log`?

Comment (JaseZiv) [reply tests/testthat/test-fbref.R]: Good call! Done

**Review by tonyelhabr — APPROVED** [review]

File: NEWS.md [hunk 5]
```diff
@@ -9,7 +9,19 @@
...
 * Fotmob functions now pass user agent to headers under the hood (0.6.0.4000)
 
+### Bugs
+
+* `tm_squad_stats()` throwing differing number of rows error [#187](https://github.com/JaseZiv/worldfootballR/issues/187) (0.6.0.5000)
+* `.get_match_report_page()` internal function used in a number of `fb_` functions throwing errors for cup tie legs [#174](https://github.com/JaseZiv/worldfootballR/issues/174) (0.6.0.5000)
+* `.clean_table_names()` change to internal function that will include '*_performance*' to some column names in `fb_team_match_log_stats()` to address duplicated column names for `stat_type = 'keeper'` [#184](https://github.com/JaseZiv/worldfootballR/issues/184) (0.6.0.5000)
+
+
+### Breaking Changes
```
Comment (tonyelhabr) [inline NEWS.md:19]: 👍

**Pull request merged**